### PR TITLE
Update T21_Generate_Death_Knight.simc

### PR DESCRIPTION
**Pull request by Maxweii**
Optimizations to Unholy profile (correct area this time)

### CODE REVIEW

File: profiles/generators/Tier21/T21_Generate_Death_Knight.simc [hunk 1]
```diff
@@ -4,9 +4,9 @@ level=110
 race=blood_elf
 role=attack
 position=back
-talents=3210031
+talents=3210011
 artifact=12:0:0:0:0:108:4:109:4:110:4:111:4:113:4:114:4:115:4:117:4:119:1:120:1:122:1:123:1:124:1:1090:4:1091:1:1092:1:1332:1:1360:1:1485:4:1486:1:1487:1:1488:24:1628:1
-crucible=1739:3:1774:3:110:2:108:1
+crucible=1739:3:1778:3:156:3
```
Comment (vituscze) [inline profiles/generators/Tier21/T21_Generate_Death_Knight.simc]: This line (and the one above) are part of the Frost profile. You probably don't want to change those.

Comment (Maxweii) [reply profiles/generators/Tier21/T21_Generate_Death_Knight.simc]: I did those by accident thought I reverted it, whoops lol

File: profiles/generators/Tier21/T21_Generate_Death_Knight.simc [hunk 2]
```diff
@@ -4,9 +4,9 @@ level=110
 race=blood_elf
 role=attack
 position=back
-talents=3210011
+talents=3210031
 artifact=12:0:0:0:0:108:4:109:4:110:4:111:4:113:4:114:4:115:4:117:4:119:1:120:1:122:1:123:1:124:1:1090:4:1091:1:1092:1:1332:1:1360:1:1485:4:1486:1:1487:1:1488:24:1628:1
-crucible=1739:3:1778:3:156:3
+crucible=1774:3:110:2:108:1
```
Comment (vituscze) [inline profiles/generators/Tier21/T21_Generate_Death_Knight.simc]: `1739:3` is the trait that adds 15 ilvls to your weapon. The original crucible string was:

    crucible=1739:3:1774:3:110:2:108:1

**Comment by Melekus** [comment]
Some items that should be at 960 are incorrectly using 970 bonus_ids.
Once this mistake is corrected the profile sims lower than the current Tier21 profile.

If you think you can still provide improvements to the profile feel free to do so !

**Pull request closed without merging**